src/Zio/FileSystems/ZipArchiveFileSystem.cs

```diff
@@ -492,12 +492,15 @@ protected override FileAttributes GetAttributesImpl(UPath path)
         var attributes = entry.FullName[entry.FullName.Length - 1] == DirectorySeparator ? FileAttributes.Directory : 0;
 
 #if NETSTANDARD2_1_OR_GREATER || NET6_0_OR_GREATER
-        if (entry.ExternalAttributes == 0 && attributes == 0)
+        const FileAttributes validValues = (FileAttributes)0x7FFF /* Up to FileAttributes.Encrypted */ | FileAttributes.IntegrityStream | FileAttributes.NoScrubData;
```

Comment:
On Linux the `entry.ExternalAttributes` was `-2119958496`, without this I couldn't make out what the attributes were...